core/src/main/java/io/questdb/cairo/TableNameRegistryRW.java

```diff
@@ -29,28 +29,27 @@
 import io.questdb.std.ObjList;
 
 public class TableNameRegistryRW extends AbstractTableNameRegistry {
-    private final ConcurrentHashMap<TableToken> nameTableTokenMap = new ConcurrentHashMap<>(false);
```

Comment:
Unnecessary refactoring

Reply:
addressed in 4231b3bb17c99d32ad3137ba2157ddd3d23c1906